EnglishFilter/sections/foreign.txt

```diff
@@ -4356,6 +4356,7 @@ dap-news.com##.ads
 !---------------------------------------------------------------------
 !
 ! https://github.com/AdguardTeam/AdguardFilters/issues/90634
+m.ygosu.com##ul#article_list > li.notice
```

Comment:
Move `m.ygosu.com###article_list > li.notice` to the AdGuard Mobile.

Reply:
https://github.com/AdguardTeam/AdguardFilters/pull/90810/commits/c2a3863e5c7362f3b3125c1ab3c9ca71b40e6b4e